web/src/pages/Classroom/EditClassroomPage/EditClassroomPage.jsx

```diff
@@ -0,0 +1,7 @@
+import EditClassroomCell from 'src/components/Classroom/EditClassroomCell'
+
+const EditClassroomPage = ({ id }) => {
+  return <EditClassroomCell id={id} />
+}
+
+export default EditClassroomPage
```

Comment:
Just commit more, have a lot of things in a single commit.

Reply:
One `Red Wood` command does a lot of changes, we didn't change much of what it has done.